src/odemis/gui/win/acquisition.py

```diff
@@ -812,16 +812,14 @@ def on_setting_change(self, _=None):
 
         wx.CallAfter(self.cmb_presets.SetValue, preset_name)
 
-    def update_acquisition_time(self):
+    def update_acquisition_time(self, acq_time=999):
         """
         Must be called in the main GUI thread.
         """
         streams = self.get_acq_streams()
 
         # TODO: Get this value from stiching
         if streams:
```

Comment:
In the latested update, there's `estimateTiledAcquisitionTime` function which takes the same parameters as `'acquireTiledArea'`, so it should be used here.